tests/test_geometry.py

```diff
@@ -0,0 +1,39 @@
+"""> PyDRex: Tests for geometric conversions and projections."""
+import numpy as np
+
+from pydrex import geometry as _geo
+
+
+def test_poles_example(example_outputs, stringify, hkl, ref_axes):
+    """Test poles (directions of crystallographic axes) of example data."""
+    ref_data = np.load(
+        example_outputs / f"example_CPO_poles_{stringify(hkl)}{ref_axes}.npz"
+    )
+    resampled_data = np.load(example_outputs / "example_CPO_resampled.npz")
+    xvals, yvals, zvals = _geo.poles(
+        resampled_data["orientations"],
+        hkl=hkl,
+        ref_axes=ref_axes,
+    )
+    np.testing.assert_allclose(ref_data["xvals"], xvals, atol=1e-16, rtol=0)
+    np.testing.assert_allclose(ref_data["yvals"], yvals, atol=1e-16, rtol=0)
+    np.testing.assert_allclose(ref_data["zvals"], zvals, atol=1e-16, rtol=0)
+
+
+def test_lambert_equal_area(rng):
+    """Test Lambert equal area projection."""
+    a = [x / 5.0 for x in range(-5, 6)]
+    x = [[x] * len(a) for x in a]
+    y = [a for _ in a]
+    x_flat = [j for i in x for j in i]
+    y_flat = [j for i in y for j in i]
```

Comment:
Surely this can be achieved with `numpy.tile` and `numpy.repeat`, given that you import `numpy`.

Reply:
Actually mgrid is fine. I think I had it like this because it was initially in a docstring example but I just copied it across. Fixed now.